### ISSUE TITLE
Cult Revival Rune Runtime [PL]

### ISSUE THREAD

**Issue by GDNgit** [ParadiseSS13/Paradise#19961]
```
[2022-12-19T23:39:36] Runtime in browserOutput.dm,303: DEBUG: to_chat called with invalid message/target. Message: '<span class='biggerdanger'>Your physical form has been taken over by another soul due to your inactivity! Ahelp if you wish to regain your form.</span>'. Target: 'Amanda Woodsworth', /datum/mind.
[2022-12-19T23:39:36]   proc name: to chat (/proc/to_chat)
[2022-12-19T23:39:36]   usr: *REDACTED USER*
[2022-12-19T23:39:36]   usr.loc: The engraved floor (21,206,1) (/turf/simulated/floor/engine/cult)
[2022-12-19T23:39:36]   src: null
[2022-12-19T23:39:36]   call stack:
[2022-12-19T23:39:36]   to chat(Amanda Woodsworth (/datum/mind), "<span class=\'biggerdanger\'>Y...", null)
[2022-12-19T23:39:36]   the rune (/obj/effect/rune/raise_dead): invoke(/list (/list))
```
Looks like by using .mind it's returning the actual datum/mind rather than the mob getting revived

**Comment by Contrabang** [comment]
https://github.com/ParadiseSS13/Paradise/blob/d1f5af4a4dbd8232ae858605b8113113d4e93463/code/game/gamemodes/cult/runes.dm#L637-L640
mob_to_revive.mind needs to become mob_to_revive, seems like an easy fix, just needs some testing

**Comment by Contrabang** [comment]
fixed by #22673
